### ISSUE TITLE
`'title': undefined method 'inject' for nil:NilClass (NoMethodError)`

### ISSUE THREAD

**Issue by moloch--** [emoriarty/notion_to_md#44]
```
/opt/homebrew/lib/ruby/gems/3.2.0/gems/notion_to_md-2.2.1/lib/notion_to_md/page.rb:13:in `title': undefined method `inject' for nil:NilClass (NoMethodError)

      page.dig(:properties, :Name, :title).inject('') do |acc, slug|
                                          ^^^^^^^
        from /opt/homebrew/lib/ruby/gems/3.2.0/gems/jekyll-notion-2.3.2/lib/jekyll-notion/generators/page_generator.rb:16:in `make_page'
        from /opt/homebrew/lib/ruby/gems/3.2.0/gems/jekyll-notion-2.3.2/lib/jekyll-notion/generators/page_generator.rb:8:in `generate'
        from /opt/homebrew/lib/ruby/gems/3.2.0/gems/jekyll-notion-2.3.2/lib/jekyll-notion/generator.rb:67:in `block in read_notion_pages'
        from /opt/homebrew/lib/ruby/gems/3.2.0/gems/jekyll-notion-2.3.2/lib/jekyll-notion/generator.rb:65:in `each'
        from /opt/homebrew/lib/ruby/gems/3.2.0/gems/jekyll-notion-2.3.2/lib/jekyll-notion/generator.rb:65:in `read_notion_pages'
        from /opt/homebrew/lib/ruby/gems/3.2.0/gems/jekyll-notion-2.3.2/lib/jekyll-notion/generator.rb:14:in `generate'
```

**Comment by emoriarty** [comment]
The has been merge into master. The fix will be available in the next release (2.2.3).

**Comment by emoriarty** [comment]
Voilà ! The release 2.2.3 is already available. I'm closing the issue.
